### ISSUE TITLE
Python library: backports.ssl_match_hostname still missing

### ISSUE THREAD

**Issue by i-s-o-g-r-a-m** [rethinkdb/rethinkdb#5470]
Hi,

The issue described here

    https://github.com/rethinkdb/rethinkdb/issues/4683

is still happening:

```
$ pip install rethinkdb
Collecting rethinkdb
  Using cached rethinkdb-2.2.0.post5.tar.gz
Installing collected packages: rethinkdb
  Running setup.py install for rethinkdb
  ...
Successfully installed rethinkdb-2.2.0.post5

$ python
Python 2.7.5 (default, Mar  9 2014, 22:15:05) 
[GCC 4.2.1 Compatible Apple LLVM 5.0 (clang-500.0.68)] on darwin
>>> import rethinkdb
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/.../site-packages/rethinkdb/__init__.py", line 3, in <module>
    from .net import *
  File "/Users/.../site-packages/rethinkdb/net.py", line 27, in <module>
    from backports.ssl_match_hostname import match_hostname, CertificateError
ImportError: No module named backports.ssl_match_hostname
>>>
```

One can install backports manually, but I'm guessing this can be fixed.

Thank you!

**Comment by danielmewes** [comment]
Thanks for reporting this @i-s-o-g-r-a-m .

This is pretty weird. I just checked https://pypi.python.org/packages/source/r/rethinkdb/rethinkdb-2.2.0.post5.tar.gz#md5=b7df81b6f8cdcb72e469cda32530ae0f and it does contain the `backports/ssl_match_hostname` subfolder.

Pinging @larkost .

**Comment by larkost** [comment]
I think I have this nailed: we are missing the `__init__.py` file from `rethinkdb/backports` because that is not defined as a module in our `setup.py`. Newer versions of Python (e.g.: Python 2.7.10 on OS X 10.11) don't care about this file, so long as there is an `__init__.py` in `rethinkdb/backports/ssl_match_hostname`.

My testing versions of Python 2.6 did not pick up on this because `backports.ssl_match_hostname` was installed by other modules I need for testing, hiding this error from me. This should be fixed in `next` with 346a6f64 and `v2.2.x` in 047184e4.
